specification/subscription/resource-manager/readme.md

```diff
@@ -33,7 +33,17 @@ These are the global settings for the Subscription API.
 openapi-type: arm
 tag: package-2018-11-preview
```

Comment:
Do you want the new tag `package-2019-03-preview` to be the default?  If so please update it here.

Reply:
Updated the tag